OakCityV2/TableViewController.m

```diff
@@ -0,0 +1,216 @@
+//
+//  TableViewController.m
+//  OakCityV2
+//
+//  Created by Josh Green on 6/8/15.
+//  Copyright (c) 2015 Josh Green. All rights reserved.
+//
+
+#import "TableViewController.h"
+#import "AFNetworking/AFNetworking.h"
+#import "DetailViewController.h"
+#import "DetailCell.h"
+#import "Job.h"
+#import "AFNetworking/UIImageView+AFNetworking.h"
+#define LANGUAGE @"PHP"
+#define TOWN @"San+Francisco"
+
+@interface TableViewController ()
+
+@end
+
+@implementation TableViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    //Fetch JSON
+    NSString *urlAsString = [NSString stringWithFormat:@"https://jobs.github.com/positions.json?description=%@&location=%@", LANGUAGE, TOWN];
+    NSURL *url = [NSURL URLWithString:urlAsString];
+    NSURLRequest *request = [NSURLRequest requestWithURL: url];
+    AFHTTPRequestOperation *operation = [[AFHTTPRequestOperation alloc] initWithRequest:request];
+    operation.responseSerializer = [AFJSONResponseSerializer serializer];
+    
+    //Parse JSON
+    [operation setCompletionBlockWithSuccess:^(AFHTTPRequestOperation *operation, id responseObject) {
+        self.jobs  = [NSMutableArray new];
```

Comment:
When you switch self.jobs back to immutable, you'll need to change this to a temporary NSMutableArray.  Let's call that 'tmpArray'.